exercise/permissions.py

```diff
@@ -16,7 +16,7 @@
 
 
 class ExerciseVisiblePermission(ObjectVisibleBasePermission):
-    message = _("Permission denied by exercise visibility")
+    message = _("Permission denied by exercise visibility.")
```

Comment:
Näistä tulee "Permission denied by exercise visibility.: The exercise is only for registered users.", joten pisteet ei oikein toimi. joko ei pistettä tai error_msg:lle parametri `delim=' '`